qa/automation/Alleva 2405/Sign_in_to_app_with_Google_account.feature

```diff
@@ -0,0 +1,132 @@
+Feature: 1.1. Sign in to App with Google account
+
+Background:
+	Given user is on https://qa.ddso-spot.quantori.com/ page
+	And user's Google account exists with valid credentials:
+		| Email                 | Password | Phone         |
+		| yana.aleeva@gmail.com | Abcd1234 | +393200238290 |
+
+Scenario: 1.1_01 Sign in with Empty Email
+	Given no user's Google account is registered on device
+	When user hits "Sign in with Google" button
+	And user leaves "Email or phone" text field empty
+	And user clicks "Next" button
+	Then �! Enter an email or phone number� error message appears under "Email or phone" text field marked red
+
+Scenario: 1.1_02 Sign in with Google account with valid credentials
+	Given no user's Google account is registered on device
+	When user hits "Sign in with Google" button
+	And user fills in "yana.aleeva@gmail.com" in "Email or phone" text field
+	And user clicks "Next" button
+	And user fills in "Abcd1234" in "Enter your password" text field
+	And user clicks "Next" button
+	Then user is signed in to app
+	And user is taken to "Dataset" page of the app
+
+Scenario: 1.1_03 Sign in with valid Password swapped with Email
+	Given no user's Google account is registered on device
+	When user hits "Sign in with Google" button
+	And user fills in "Abcd1234" in "Email or phone" text field
+	And  user clicks "Next" button
+	Then "! Couldn�t find your Google Account" error message appears under "Email or phone" text field marked red
```

Comment:
This Google account is exist, because google automatically decide that it is  "Abcd1234@gmail.com" address

Reply:
Input is "Abcd1234" not "Abcd1234@gmail.com"

Reply:
try to do it in real google autorization page

Reply:
Good catch, Scenario would have to be rewritten completely, but now is irrelevant as I fully changed initial scenarios due t your comment on "what we are testing"